registry/storage/driver/s3-aws/s3.go

```diff
@@ -331,6 +332,8 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		objectACL = objectACLString
 	}
 
+	sessionToken := ""
```

Comment:
@dmcgowan, let me know if you prefer to inline this but assigning a variable makes the struct initialization read better, I think.